common/types/timestamp.go

```diff
@@ -80,15 +82,29 @@ func (t Timestamp) Compare(other ref.Val) ref.Val {
 
 // ConvertToNative implements ref.Val.ConvertToNative.
 func (t Timestamp) ConvertToNative(typeDesc reflect.Type) (interface{}, error) {
-	if typeDesc == timestampValueType {
+	switch typeDesc {
+	case anyValueType:
+		return ptypes.MarshalAny(t.Timestamp)
+	case jsonValueType:
+		// proto3 to JSON conversion requires string-formatted timestamps
```

Comment:
We follow proto3 to JSON conversion which formats as an RFC 3339 encoded JSON string.